kpi/views.py

```diff
@@ -514,6 +514,12 @@ def preview(self, request, *args, **kwargs):
                                       request=request,
                                       ),
                             )
+            deployment_data = snapshot.asset._deployment_data
+            if 'backend_response' in deployment_data:
+                preview_url += '{}/xformsManifest/{}'.format(
```

Comment:
Based on enketo-express#643, I'm guessing this is supposed to pass `&manifestUrl=` in the preview_url value